xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -72,14 +73,16 @@ private static Schema newSchema(Source[] schemas) throws SAXException {
     }
   }
 
-  private static final URL CORE_SCHEMA_URL = XmlConfiguration.class.getResource("/ehcache-core.xsd");
+  public static final URL CORE_SCHEMA_URL = XmlConfiguration.class.getResource("/ehcache-core.xsd");
```

Comment:
I suspect there is some super-type we can pull out of this and the ConfigurationTranslator. It also seems weird that the translator has an unmarshaller, and the parser has a marshaller.

Reply:
The marshaller and unmarshaller are there for the `ResourceConfigurationParser`.  But I guess I can have two different constructors for that class that take the marshaller and unmarshaller separately instead of both.